superset/jinja_context.py

```diff
@@ -335,8 +421,8 @@ def get_template_processor(
     **kwargs: Any,
 ) -> BaseTemplateProcessor:
     if feature_flag_manager.is_feature_enabled("ENABLE_TEMPLATE_PROCESSING"):
-        template_processor = template_processors.get(
-            database.backend, BaseTemplateProcessor
+        template_processor = template_processors().get(
+            database.backend, JinjaTemplateProcessor
```

Comment:
nit: I'd personally either prefer `@property` on `template_processors()` or rename to `get_template_processors()`.

Reply:
I'm no Python expert, but it doesn't appear that `@property` can be used outside of a class context.

Reply:
LGTM as is, if anything it can be done outside of this PR

Reply:
Whoops, forgot we were outside class context 🤦